mkdocs.yml

```diff
@@ -44,6 +44,7 @@ nav:
     - External Secrets: 'addons/external-secrets.md'
     - EMR on EKS: 'addons/emr-eks.md'
     - FluxCD: 'addons/fluxcd.md'
+    - GrafanaOperator: 'addons/grafana-operator.md'
```

Comment:
This should be a human readable name so Grafana Operator (two words)

Reply:
Fixed